src/node/clientsignatures.h

```diff
@@ -34,55 +42,8 @@ namespace ccf
 
     MSGPACK_DEFINE(sig, req, request_body, md);
   };
+  DECLARE_JSON_TYPE(SignedReq)
+  DECLARE_JSON_REQUIRED_FIELDS(SignedReq, sig, req, request_body, md)
   // this maps client-id to latest SignedReq
   using ClientSignatures = Store::Map<CallerId, SignedReq>;
-
-  inline void to_json(nlohmann::json& j, const SignedReq& sr)
```

Comment:
This was unused and actually wrong since the FTCP signature over msgpack-serialised request was removed.